hackathon/views.py

```diff
@@ -1,10 +1,61 @@
 from django.views.generic import ListView
+from django.shortcuts import render, get_object_or_404, redirect
+from django.contrib import messages
+from django.contrib.auth.decorators import login_required
+from datetime import datetime
```

Comment:
`datetime`  is a default module and as per PEP8 should be imported before the 3rd party modules (django modules in this case) and separated by a blank line.

Reply:
Didn't know that! I've moved it to the top of the file and separated with a blank line. This change will be visible once pushed to the existing PR.